webapp/app/[locale]/stake/layout.tsx

```diff
@@ -1,69 +1,11 @@
-'use client'
+import { Metadata } from 'next'
 
-import { StakeTabs } from 'components/stakeTabs'
-import { useNetworkType } from 'hooks/useNetworkType'
-import { useStakeTokens } from 'hooks/useStakeTokens'
-import dynamic from 'next/dynamic'
-import { Suspense } from 'react'
-import { isStakeEnabledOnTestnet } from 'utils/stake'
+import StakeLayoutClient from './stakeLayoutClient'
```

Comment:
My only comment: I'd move this file into `_components` folder here 

```suggestion
import StakeLayoutClient from './_components/stakeLayoutClient'
```

So we keep the `stake` folder clean. After all, the layout file is this one (`layout.tsx`). The other becomes a component 😅